src/fetch_foursquare_info.py

```diff
@@ -0,0 +1,129 @@
+import os.path
+import requests
+import numpy as np
+import pandas as pd
+import re
+import json
+import datetime
+
+# Setting dataset paths
+REIMBURSEMENTS_DATASET_PATH = os.path.join('data', '2016-11-22-reimbursements.xz')
+COMPANIES_DATASET_PATH = os.path.join('data', '2016-09-03-companies.xz')
+FOURSQUARE_DATASET_PATH = os.path.join('data', 'foursquare-companies.json')
```

Comment:
Can we save data in the standard format for the project? CSV compressed as `.xz`?